test/Dapr.AspNetCore.Test/DaprClientBuilderTest.cs

```diff
@@ -44,15 +44,15 @@ public void DaprClientBuilder_UsesPropertyNameCaseHandlingAsSpecified()
         public void DaprClientBuilder_UsesThrowOperationCanceledOnCancellation_ByDefault()
         {
             var builder = new DaprClientBuilder();
-            var daprClient = builder.Build();
+            builder.Build();
```

Comment:
These lines can be combined. 

```csharp 
var builder = new DaprClientBuilder().Build();
```

Reply:
Thank you! Addressed in my latest commit!